uber/models.py

```diff
@@ -1025,6 +1035,7 @@ class Attendee(MagModel, TakesPaymentMixin):
     placeholder   = Column(Boolean, default=False, admin_only=True)
     first_name    = Column(UnicodeText)
     last_name     = Column(UnicodeText)
+    legal_name_   = Column('legal_name', UnicodeText, nullable=True, default=None)
```

Comment:
I was wondering about this as well. I thiiiiink if this were me, I would go with:
`legal_name` as the column name to fulfill the second/third use-cases
`legal_or_full_name` as a hybrid property that fulfills the first use-case

This is most consistent with how we handle other cases like this (e.g., `ribbon_and_or_badge_type`) and I think when used in the code it'll be the most readable (at least, I have no idea what `legal_name_opt` means without any context).